docs/user_guide/configuring.rst

```diff
@@ -123,6 +121,49 @@ As the Sphinx theme supports multiple modes, the code highlighting colors can be
 
    The native Sphinx option `pygments_style` will be overwritten by this theme.
 
+Announcement banners
+====================
+
+You can add an announcement banner that draws extra attention from your reader.
+It will be displayed at the top of the screen, but will disappear once you start scrolling.
+
+To add an announcement banner, use the ``html_theme_options.announcement`` configuration.
```

Comment:
shouldn't these be `html_theme_options['announcement']` (rather than the dot syntax)?  It is a python dictionary after all.